packages/functional-tests/tests/react-conversion/signup.spec.ts

```diff
@@ -167,12 +167,10 @@ test.describe('severity-1 #smoke', () => {
 
       await signupReact.fillOutCodeForm(code);
 
-      // TODO code verification is currently throwing an UNAUTHORIZED error
-      // remove the following line once this error is no longer thrown
-      // expect(await page.getByText(/Invalid token/).isVisible()).toBeFalsy();
-      // TODO uncomment these lines once line above is passing (no token error)
-      // await page.waitForURL(/connect_another_device/);
-      // expect(await page.getByText('Youʼre signed into Firefox').isVisible()).toBeTruthy();
+      // See note in `firefox.ts` about an event listener hack needed for this test
+      expect(await page.getByText(/Invalid token/).isVisible()).toBeFalsy();
```

Comment:
Not sure if you need to assert invalid token text

Reply:
Yeah you're right, the navigation to the next page means it was successful. I'll remove this with my follow up PR with tests for some of this.